tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+    },
+    "identity": {
+        "symbol": "id_A",
+        "alt_symbol": "1_A",
+        "meaning": "mandatory identity arrow on object A"
+    },
+    "functor": {
+        "symbol": "F: C → D",
+        "meaning": "structure-preserving map between categories C and D"
+    },
+    "natural_transformation": {
+        "symbol": "α: F ⇒ G",
```

Ruff (0.16.2)

[warning] String contains ambiguous `α` (GREEK SMALL LETTER ALPHA). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)